src/clint.py

```diff
@@ -2547,22 +2547,23 @@ def CheckStyle(filename, clean_lines, linenum, error):
     # if(match(prev, " +for \\(")) complain = 0;
     # if(prevodd && match(prevprev, " +for \\(")) complain = 0;
     initial_spaces = 0
-    cleansed_line = clean_lines.elided[linenum]
+    # cleansed_line = clean_lines.elided[linenum]
 
     while initial_spaces < len(line) and line[initial_spaces] == ' ':
         initial_spaces += 1
 
-    if (cleansed_line.count(';') > 1 and
-        # for loops are allowed two ;'s (and may run over two lines).
-        cleansed_line.find('for') == -1 and
-        (GetPreviousNonBlankLine(clean_lines, linenum)[0].find('for') == -1 or
-         GetPreviousNonBlankLine(clean_lines, linenum)[0].find(';') != -1) and
-        # It's ok to have many commands in a switch case that fits in 1 line
```

Comment:
looks like clint already tried to allow multiple things on one line.

Anyways, prefer to delete rather than comment-out client rules. We want clint.py to be as small as possible.